cpg-language-cxx/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/cxx/CXXLanguageFrontendTest.kt

```diff
@@ -1758,4 +1757,23 @@ internal class CXXLanguageFrontendTest : BaseTest() {
         assertIs<CastExpression>(cast)
         assertLocalName("mytype", cast.castType)
     }
+
+    @Test
+    @Throws(Exception::class)
```

Comment:
The `Throws` shouldn't be required, right?

Reply:
No, it's not. I just copied the above test to keep the same pattern. -> Removed.